tests/bench/bench.ml

```diff
@@ -99,6 +99,39 @@ let () =
     launch (fmt "%s switch create six --empty" bin);
     time_cmd ~exit:0 (fmt "%s list --installed --short --safe --color=never ocp-indent ocp-index merlin" bin)
   in
+  launch (fmt "%s switch create seven --empty" bin);
+  launch (fmt "%s install -y --fake dune" bin);
+  let time_show_installed =
+    Gc.compact ();
+    time_cmd ~exit:0 (fmt "%s show dune" bin)
+  in
+  let time_show_with_depexts =
+    Gc.compact ();
+    time_cmd ~exit:0 (fmt "%s show conf-llvm" bin)
+  in
+  let time_show_raw =
+    Gc.compact ();
+    time_cmd ~exit:0 (fmt "%s show --raw conf-llvm" bin)
+  in
+  let time_show_precise =
+    Gc.compact ();
+    time_cmd ~exit:0 (fmt "%s show --raw conf-llvm.14.0.6" bin)
+  in
+  let time_OpamStd_String_split =
+    Gc.compact ();
+    let lines =
+      let ic = Stdlib.open_in_bin "/home/opam/all-opam-content" in
+      let rec loop files =
+        match Stdlib.input_line ic with
+        | file -> loop (file :: files)
+        | exception End_of_file -> files
+      in
+      loop []
+    in
+    let before = Unix.gettimeofday () in
+    List.iter (fun line -> ignore (OpamStd.String.split line ' ')) lines;
```

Comment:
Wouldn't be more precise if it is run 50, 100 times ? Current run/time is less than 1 second.

Reply:
the problem is that it is only 1s in #6210. In this PR it takes 45s which would be too long to run 50 or 100 times.

Reply:
i can run it 10 times maybe? That would be 4.5mins for this benchmark which seems still reasonable

Reply:
yes, the idea was to not have to diff times under 1s, to have more "accurate" data.

Reply:
The 5min bench is just for few launches

Reply:
done